features/pom.xml

```diff
@@ -35,68 +39,116 @@
-            <groupId>org.apache.karaf.features</groupId>
-            <artifactId>framework</artifactId>
-            <type>kar</type>
-            <scope>runtime</scope>
-            <version>${karaf-version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.karaf.features</groupId>
-            <artifactId>standard</artifactId>
-            <classifier>features</classifier>
-            <type>xml</type>
-            <scope>runtime</scope>
-            <version>${karaf-version}</version>
-        </dependency>
-
-    </dependencies>
+    <profiles>
+        <profile>
+            <id>verify-feature</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.karaf.tooling</groupId>
+                        <artifactId>karaf-maven-plugin</artifactId>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <enableGeneration>true</enableGeneration>
+                            <aggregateFeatures>true</aggregateFeatures>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>feature-verify</id>
+                                <phase>process-resources</phase>
+                                <goals>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <descriptors>
+                                        <descriptor>mvn:org.apache.karaf.features/framework/${karaf-version}/xml/features</descriptor>
+                                        <descriptor>mvn:org.apache.karaf.features/standard/${karaf-version}/xml/features</descriptor>
+                                        <descriptor>mvn:org.apache.karaf.features/enterprise/${karaf-version}/xml/features</descriptor>
+                                        <descriptor>mvn:org.apache.karaf.features/spring/${karaf-version}/xml/features</descriptor>
+                                        <descriptor>file:${project.build.directory}/feature/feature.xml</descriptor>
+                                    </descriptors>
+                                    <distribution>org.apache.karaf.features:framework</distribution>
+                                    <javase>11</javase>
+                                    <framework>
+                                        <feature>framework</feature>
+                                        <feature>shell-compat</feature>
+                                    </framework>
+                                    <features>
+                                        <feature>tinybundles</feature>
+                                    </features>
```

Comment:
Check the JOOQ project features please. To be able to verify maybe some JDBC feature will be required. (Maybe karaf-features-jdbc have to be added ?)

Reply:
True. I updated my mistake from the jooq-features pom.xml.